### PR TITLE
[INLONG-8073][DataProxy] Add HTTP message processing logic in source2

### LINKED ISSUE #8073: [Improved][DataProxy]Add HTTP message processing logic in source2
### Description

Add message processing support for Http reporting in source2. The difference from org.apache.inlong.dataproxy.http using javax.servlet.http is that source2 uses io.netty to support HTTP request processing

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
- Fixes #8073

1. Add HTTP processing handler
2. Adjust the current implementation and merge public functions
3. Adjust the order in which each class file references the package

### CODE REVIEW

**Review by luchunliang — APPROVED** [review]
LGTM

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/InLongMessageHandler.java [hunk 1]
```diff
@@ -44,30 +62,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.nio.charset.StandardCharsets;
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/InLongMessageHandler.java:47]: It seems that your code style in your local IDE was changed.

Please refer to this: https://github.com/apache/inlong/discussions/3082 and reformat the import order, thanks.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/SimpleUdpSource.java [hunk 2]
```diff
@@ -17,15 +17,15 @@
 
 package org.apache.inlong.dataproxy.source2;
 
+import java.net.InetSocketAddress;
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/SimpleUdpSource.java:20]: ditto.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/httpMsg/InLongHttpMsgHandler.java [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.dataproxy.source2.httpMsg;
+
+import static io.netty.handler.codec.http.HttpUtil.is100ContinueExpected;
+
+import java.io.UnsupportedEncodingException;
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/httpMsg/InLongHttpMsgHandler.java:22]: ditto.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/v0msg/AbsV0MsgCodec.java [hunk 4]
```diff
@@ -30,12 +34,6 @@
 import org.apache.inlong.dataproxy.utils.DateTimeUtils;
 import org.apache.inlong.dataproxy.utils.InLongMsgVer;
 
-import java.nio.charset.StandardCharsets;
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/v0msg/AbsV0MsgCodec.java:33]: ditto.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/v0msg/CodecBinMsg.java [hunk 5]
```diff
@@ -17,26 +17,6 @@
 
 package org.apache.inlong.dataproxy.source2.v0msg;
 
-import org.apache.commons.lang3.StringUtils;
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/v0msg/CodecBinMsg.java:20]: ditto.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/SimpleTcpSource.java [hunk 6]
```diff
@@ -28,13 +30,11 @@
 import org.apache.inlong.dataproxy.config.ConfigManager;
 import org.apache.inlong.dataproxy.config.holder.ConfigUpdateCallback;
 import org.apache.inlong.dataproxy.utils.AddressUtils;
+import org.apache.inlong.dataproxy.utils.ConfStringUtils;
 import org.apache.inlong.dataproxy.utils.EventLoopUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.net.InetSocketAddress;
```
[the remark this answers is not in the document]
Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source2/SimpleTcpSource.java:35]: done

**Review by EMsnap — APPROVED** [review]

**Pull request merged**